tools/packaging/packaging.psm1

```diff
@@ -318,6 +353,19 @@ function New-TarballPackage {
     }
 }
 
+function New-TempFolder
+{
+    $tempPath = [System.IO.Path]::GetTempPath()
+
+    $tempFolder = Join-Path -Path $tempPath -ChildPath ([System.IO.Path]::GetRandomFileName())
```

Comment:
How about replace these 2 lines with `[System.IO.Path]::GetTempFileName()`?

Reply:
This API creates the file.  I want a folder.

Reply:
I know, but you can use that name to create a folder. The two lines of script shown here are doing the same thing.

Reply:
I know, but you can use that name to create a folder. The two lines of script shown here are doing the same thing.

Reply:
I would have to delete the file before I can create a folder with the name.

Reply:
I see your point. Yeah, please ignore this comment.